Sources/Session.php

```diff
@@ -96,6 +96,11 @@ public function read(string $session_id): string
 	 */
 	public function write(string $session_id, string $data): bool
 	{
+
+		// Don't bother writing the session if cookies are disabled
+		if (empty($_COOKIE))
+			return true;
```

Comment:
This makes PHP-CS-Fixer sad. Please change to:

```suggestion
		if (empty($_COOKIE)) {
			return true;
		}
```

Reply:
I'll take your word for it!  I figured I passed php-cs-fixer given the green checkmark on the PR.

I was never able to get php-cs-fixer to work acceptably locally.  I have it installed, but it seems to want to do very strange stuff.  E.g., in this example, it changed 90% of tabs to spaces.  (Not all of them...)  It did add the braces, though.

No plugins for the tools I use either (Notepad++, Eclipse).  I can integrate by hand, but I get the same odd results, e.g., lots of tabs become spaces.   So I don't use it.

Reply:
Shouldn't have to install it if composer has ran
```
composer lint
composer lint-fix
```

I also added it to my git as an alias
```
        phpcs = "!f() { ./vendor/bin/php-cs-fixer --config=.php-cs-fixer.dist.php list-files | xargs -n 50 -P 8 ./vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.dist.php --path-mode intersection -v; }; f"```
```
Note that the 8 is concurrent threads.

I started to test adding it to .git/hooks/pre-commit
```
#!/bin/sh
git phpcs
```
I hope to get it to run before I issue a commit, but I haven't gotten around to testing it more. I need to work on adding it when I stage the commit.

Reply:
Our instructions said to install it, so I installed it.  I had to install composer, also...  I'm not familiar with either tool.

But it did install, & it does make the changes.  But it also screws up the whitespace.  

Does it change most of your tabs to spaces also?  Or is that just a me thing?  😆

Reply:
Hmmm....  i see with your example above, you tell it to use our rules with a config override (--config=.php-cs-fixer.dist.php)....

That wasn't mentioned anywhere...  I'll experiment with that as well.

Reply:
Correct, it needs our config or it applies the defaults.  Why people like spaces over tabs is beyond me as tabs can be adjusted in the editor to favor whatever spacing you prefer.

Reply:
And yes, I just confirmed that it works.  THANX!!!

The notes over on the Development Contributors board probably should reflect this.

I wrote a little .bat file that is invoked by Notepad++ and it works just fine now.